packages/flutter_adaptive_scaffold/example/pubspec.yaml

```diff
@@ -12,9 +12,11 @@ dependencies:
     sdk: flutter
   flutter_adaptive_scaffold:
     path: ..
+  go_router:
+    path: ../../go_router
```

Comment:
This is probably OK, but  I wonder if it might be better to use a published version here.  But maybe this is required because `go_router` is in the same repo?

Reply:
I think we should pin a value here. If the future work require changes to go_router, we will create one pr to go_router first and another pr to update this package along with this pubspec.yaml.

Reply:
@chunhtai done.